cli/module_loader.rs

```diff
@@ -54,18 +129,45 @@ impl ModuleLoader for CliModuleLoader {
 
   fn load(
     &self,
-    module_specifier: &ModuleSpecifier,
+    specifier: &ModuleSpecifier,
     maybe_referrer: Option<ModuleSpecifier>,
     is_dynamic: bool,
   ) -> Pin<Box<deno_core::ModuleSourceFuture>> {
-    let module_specifier = module_specifier.clone();
-    let ps = self.ps.clone();
+    log::debug!(
```

Comment:
I feel like this log here is superfluous, I'd be fine removing it

Reply:
Sure!